src/main/resources/sessionLinksRecoveryEmailTemplate-feedbackSessionAccessLinksNone.html

```diff
@@ -0,0 +1,32 @@
+<p>Hello,</p>
+
+<p>
+    You are receiving this email because someone requested <a href="${teammateHomePageLink}">TEAMMATES</a> (an online peer evaluation system) to send access links to this email address. If you did not make that request, please ignore this email.
+</p>
+
+<p>
+    You do not have any feedback sessions for the past 180 days. If you have any doubt, please contact your course instructors to check if the email below is the one used for your course enrollment:
+</p>
+
+<p>
+    <strong>
+        ${userEmail}
+    </strong>
+</p>
+
+<p>
+    If a feedback session that you expect to see is missing, it may be associated with a different email address or
+    it is closed and result not visible for the past 180 days. You can try <a href=${sessionsRecoveryLink}>
```

Comment:
Remove this part `it is closed and result not visible for the past 180 days.` Those are not the correct conditions to omit a session. The only reason to omit a session is if the session is not yet visible to the student, as per the visiblity setting of the session.